CHANGELOG.rst

```diff
@@ -1,40 +1,49 @@
 2.9.0.dev
 =========
 
-New Features
-------------
+**New Features**
```

Comment:
The [docutils doc](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#implicit-hyperlink-targets) describes what the error means, but I don't know a workaround either...

I'm guessing `rst-lint` exits non-zero in this case (with only `INFO` messages) as well? Otherwise we might just want to ignore `INFO` messages?